core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -487,7 +488,7 @@ private static String formatFragment(
             Optional<TDigestHistogram> outputBufferUtilization = stageInfo.get().getStageStats().getOutputBufferUtilization();
             if (verbose && outputBufferUtilization.isPresent()) {
                 builder.append(indentString(1))
-                        .append(format("Output buffer utilization distribution (%%): {p01=%s, p05=%s, p10=%s, p25=%s, p50=%s, p75=%s, p90=%s, p95=%s, p99=%s, max=%s}\n",
+                        .append(format("Output buffer utilization distribution (%%): {p01=%s, p05=%s, p10=%s, p25=%s, p50=%s, p75=%s, p90=%s, p95=%s, p99=%s, max=%s, duration=%s}\n",
```

Comment:
Should the duration recording be printed separately from "Output buffer utilization distribution" ?
Maybe it can be another key-value pair with "Output buffer active/elapsed time" as the name

Reply:
I'm not sure. That would add one extra line to already verbose output

Reply:
It could be on the same line